x-pack/plugins/cases/server/attachment_framework/types.ts

```diff
@@ -14,12 +14,12 @@ export type PersistableStateAttachmentState = Pick<
 >;
 
 export interface PersistableStateAttachmentType
-  extends PersistableState<PersistableStateAttachmentState> {
+  extends Omit<PersistableState<PersistableStateAttachmentState>, 'migrations'> {
```

Comment:
@cnasikas @js-jankisalvi have any plugins registered migrations? If so, they won't work any more right?

Reply:
Hey! Fortunately, no plugin has registered any migrations so far. It is safe to remove it.